app/views/devise/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if resource.errors.any? %>
```

Comment:
we'll move this to a common error handling (with other stuff we do) but fine for now